courses/machine_learning/intro_to_tensorflow/2_dataset_api.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>Do we want to number sections/subsections? We haven't in previous labs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Removed numbering</p>